### DISCUSSION TITLE
How do I access the open link on Colab?

### DISCUSSION THREAD

**Discussion by meutim444** [Sanster/IOPaint#174] [Q&A]
How do I access the open link on Colab?

I'm stuck at this message: 

2023-01-06 21:30:29.783 | INFO     | lama_cleaner.helper:load_jit_model:44 - Load model from: /root/.cache/torch/hub/checkpoints/clickseg_pplnet.pt
INFO:werkzeug: * Running on http://0.0.0.0:4242/ (Press CTRL+C to quit)

this is the last one. No link to click like Stable Dissufion does.

**Comment by Sanster** [comment]
Open the ngrok public URL(in 2. Setup ngrok) to access the web app
